README.md

```diff
@@ -58,6 +58,7 @@ bin list # List current binaries and it's versions
 bin prune # Removes from the DB missing binaries
 bin remove <bin>... # Deletes one or more binaries
 bin update [bin]... # Scans binaries and prompts for update
+bin --debug # show config directory, download path
```

Comment:
I don't think `--debug` should be listed here as an example given that's it's a flag that applies to all commands.

Reply:
Your point is valid.

However, what other option do I have to "show config directory, download path"? 
As a new user, I had to google for it, and it was not obvious, maybe on 5th or 10th link I found a suggestion. Thus, I try to reduce learning curve for others.

Alternative would be add new option, e.g. `show config` or `info` or `--info` or add this info to `--version`.

Reply:
> Alternative would be add new _obvious_ option, e.g. `show config` or `info` or `--info` or add this info to `--version`.

 Adding a `bin config show` doesn't seem that bad. WDYT?

Reply:
`bin config show`  does sound consistent with many tools out there, and is explicit, obvious option. Would be great!

Reply:
> `bin config show` does sound consistent with many tools out there, and is explicit, obvious option. Would be great!

 awesome! any chance you're willing to open a PR to address this one?